### DISCUSSION TITLE
Legendary clown costume on female character = dress

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#6700] [Ideas and suggestions]
I am not insane and this is genuine:
when a female character uses the "Mother's Countenance" and/or "Mother's Providence" the outfit turns into a clown-themed dress.

**Comment by 3e849f2e5c** [comment]
Based

**Comment by Lithosagym** [comment]
Yeah,

**Comment by TBenz4Sportsball** [comment]
TBenz supports this clownery.

Praise the Honkmother.

**Comment by Jade-Harleyy** [comment]
agree

**Comment by ahzidaljun** [comment]
Payasus

**Comment by HeatedWafflez** [comment]
Honestly, quite incredible.

**Comment by StruggleDuck** [comment]
agreed

**Comment by WickedWolfy** [comment]
Am I the only one who foresees Harley Quin dressed female characters everywhere int he future?

**Comment by Hazmat616** [comment]
Clussy fever?
